src/jit/codegen.h

```diff
@@ -931,6 +931,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     void genHWIntrinsic_R_RM(GenTreeHWIntrinsic* node, instruction ins, emitAttr attr);
     void genHWIntrinsic_R_RM_I(GenTreeHWIntrinsic* node, instruction ins, int8_t ival);
     void genHWIntrinsic_R_R_RM(GenTreeHWIntrinsic* node, instruction ins, emitAttr attr);
```

Comment:
Might it be worthwhile to just get rid of this one and have the callsites updated to call the new overload directly?

Reply:
The original overload is called by several places, that would duplicate some code without the wrapper overload. I will refactor some HW intrinsic code later, that may be a good time to reconsider this.